### PR TITLE
Support for 2nd Prometheus Snapshot

### PR DESCRIPTION
**Pull request by tjungblu**
This PR adds a little toggle to select the second instance snapshot (if available) from the gcs bucket.

Apologies for the diff, my IDE formatted the jsx file for me :/ I can reset this if you fancy.

### CODE REVIEW

File: pkg/promecieus/helpers.go [hunk 1]
```diff
@@ -76,18 +77,18 @@ func getLinksFromURL(url string) ([]string, error) {
 	}
 }
 
-func ensureMetricsURL(url string) (int, error) {
+func ensureMetricsURL(url *url.URL) (int, error) {
 	var netClient = &http.Client{
```
Comment (vrutkovs) [inline pkg/promecieus/helpers.go:84]: Lets add a check that `url` is not nil

**Review by vrutkovs — CHANGES_REQUESTED** [review]

File: pkg/promecieus/helpers.go (hunk 1 above)
Comment (tjungblu) [reply pkg/promecieus/helpers.go:84]: done

**Comment by vrutkovs** [comment]
Thank you! Deployed this to production instance

**Comment by tjungblu** [comment]
Awesome! Thanks :) Can you also review the other PR? https://github.com/openshift/release/pull/37114 I believe you're in the owners file

**Pull request merged**